test/transfer-asset.spec.ts

```diff
@@ -29,10 +29,10 @@ test("TransferAsset - success", async () => {
     });
     await Helper.sdk.key.signTransactionInput(transferAsset, 0);
     const { block } = await Helper.sendTransactionAndGetBlock(transferAsset);
-    await BlockModel.createBlock(block, Helper.sdk, new U64(1), {});
+    await BlockModel.createBlock(block, Helper.sdk, new U64(1));
 });
 
-test("TransferAsset - Nonexistent assetType", async () => {
+test.skip("TransferAsset - Nonexistent assetType", async () => {
```

Comment:
I skipped these tests. CodeChain doesn't include the failed transactions anymore.